app/dependencies.php

```diff
@@ -127,13 +128,23 @@ function (ContainerInterface $c): Auth\DonationPublicAuthMiddleware {
             $messengerReceiverLocator->set($priorityKey, $c->get(Transports::TRANSPORT_HIGH_PRIORITY));
             $messengerReceiverLocator->set($salesforceKey, $c->get(Transports::TRANSPORT_LOW_PRIORITY));
 
+            $eventDispatcher = new EventDispatcher();
+
+            $eventDispatcher->addListener(WorkerMessageReceivedEvent::class, function () use ($c) {
+                // clear the entity manager before handling each message to make sure we get up-to-date copies
+                // of any entities - otherwise they could be up to an hour out of date as the consumer is a long-
```

Comment:
I'm not sure I see where one hour comes from? Typical workers currently live up to 24 hours but maybe there's a cache lifetime default I'm missing?

Reply:
You're right it is 24 hours, I somehow read saw 86400 here and thought of 3600 instead. https://github.com/thebiggive/matchbot/blob/95f59cc97c66c80dc7bc821eb536d2e9b962c6fb/composer.json#L217

Reply:
Fixed in [64fd062](https://github.com/thebiggive/matchbot/pull/1288/commits/64fd0628a0f3c15ab658eb2c14ae981be87e4ad3) , thanks @NoelLH